recipes/patch-ng/LICENSE

```diff
@@ -0,0 +1,53 @@
+MIT License
```

Comment:
Can you please make a PR to get this included in the upstream sdist?

Reply:
Good one, I've created a PR for this:
https://github.com/conan-io/python-patch-ng/pull/7